src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -425,16 +430,23 @@ String mafTransform(final String key, final String value) {
                 if ( value.equals(MafOutputRendererConstants.FieldValue_Gencode_Chromosome_Mito) ) {
                     return MafOutputRendererConstants.FieldValue_Chromosome_Mito;
                 }
-                else if ( value.toLowerCase().startsWith("chr") ) {
-                    return value.substring(3);
+                else if ( value.toLowerCase().startsWith("chr") && (referenceVersion.toLowerCase().equals("hg19") || referenceVersion.toLowerCase().equals("b37"))) {
```

Comment:
What is this doing?  Can't we get this from the reference?  There are other contigs that can show up in a MAF besides 1-22, X, and Y.

Also, why don't you use a set and ask if it contains `trimVal`?

Reply:
This is making it so that only the contig names that need to get changed are the ones that get changed on output.  Before it would remove "chr" from the start of any contig regardless of the contig name or reference.  Since this seems to be an hg19/b37 specific case, and specific to only the numbered contigs plus X, Y, and M, I wanted to make it as specific as possible.

This directly fell out of my needing to create a direct comparison between MAF and VCF files for a test.  When I was instrumenting the `mafTransformInverse` method I had to do something similar and I realized this code was incorrect.

I don't have direct access to the reference where this is being called.

I'm specifically not changing those contig values because they don't need to change.

I will update the check with a set.

Reply:
ok... I guess that is fine